scripts/ci/credscan/CredScanSuppressions.json

```diff
@@ -573,6 +573,26 @@
         "src\\azure-cli\\azure\\cli\\command_modules\\acs\\tests\\latest\\data\\setup_proxy.sh"
       ],
       "_justification": "Dummy self-signed certificate + private key used for testing only."
+    },
+    {
+      "placeholder": "\"sharedKey\":\"abc123\"",
+      "_justification": "[containerapp] request body contains sharedKey recognized as secret"
+    },
+    {
+      "placeholder": "\"primarySharedKey\":\"abc123\"",
+      "_justification": "[containerapp] request body contains sharedKey recognized as secret"
```

Comment:
Is it the same as above? Maybe it should be `primarySharedKey` instead of `sharedKey`.

Reply:
The `sharedKey `is our property for create Containerapp environment.

`"logAnalyticsConfiguration": {"customerId": "f55f459c-8354-4c43-8a56-bd0189f7d8c2", "sharedKey":"abc123"}`